simplelink/source/ti/devices/cc13x2_cc26x2/CMakeLists.txt

```diff
@@ -19,6 +19,12 @@ zephyr_library_sources(
   driverlib/driverlib_release.c
   )
 
+# Required for IEEE 802.15.4 support
+zephyr_sources_ifdef(CONFIG_IEEE802154_CC13XX_CC26XX
```

Comment:
Ah. I just saw this now.
I'll also need rf_patch_cpe_bt5.c to be linked in for my BLE changes. Made a separate PR for that in #17

Reply:
I guess, technically, there is no harm in adding a Kconfig option to Zephyr that does nothing (yet). CONFIG_BLE_CC13XX_CC26XX is defined in [my branch](https://github.com/cfriedt/zephyr/blob/simplelink-ble-for-zephyr/soc/arm/ti_simplelink/cc13x2_cc26x2/Kconfig.defconfig.series#L64).

Reply:
Yes a Kconfig variable for BLE would be necessary to support conditional compilation here.